best-practices/tech_lead.md

```diff
@@ -0,0 +1,367 @@
+# Technical Lead
```

Comment:
Should this be infrastructure team only?

Reply:
Very likely, yes, IMO.